### PR TITLE
Improve Venus cards: Skip selection if only one possible target

### PR DESCRIPTION
**Pull request by nwai90**
**Scope:** Corroder Suits, Floating Habs, Maxwell Base, Venus Soils

Will address **Thermophiles** separately after https://github.com/bafolts/terraforming-mars/pull/802 is merged

### CODE REVIEW

File: src/cards/venusNext/FloatingHabs.ts [hunk 1]
```diff
@@ -31,6 +31,31 @@ export class FloatingHabs implements IActionCard,IProjectCard, IResourceCard {
...
+              if (htp.heat + htp.megaCredits < 2) {
+                  throw new Error('Not enough spent for action');
+              }
+              player.megaCredits -= htp.megaCredits;
+              player.heat -= htp.heat;
+              player.addResourceTo(floaterCards[0], 1);
+              return undefined;
+            }
+          );
+        }
```
Comment (vincentneko) [inline src/cards/venusNext/FloatingHabs.ts:52]: Please use these lines instead:
game.addSelectHowToPayInterrupt(game, 2, false, false, "Select how to pay for Floating Habs action");
return undefined;

Comment (nwai90) [reply src/cards/venusNext/FloatingHabs.ts]: 👌 @vincentneko I updated

File: src/cards/venusNext/FloatingHabs.ts [hunk 2]
```diff
@@ -29,30 +29,32 @@ export class FloatingHabs implements IActionCard,IProjectCard, IResourceCard {
...
+    public action(player: Player, game: Game) {
       const floaterCards = player.getResourceCards(ResourceType.FLOATER);
+
+      // add to itself if no other available target
+      if (floaterCards.length === 1) {
+        if (player.canUseHeatAsMegaCredits && player.heat > 0) {
+          game.addSelectHowToPayInterrupt(player, 2, false, false, "Select how to pay for Floating Habs action");
+        } else {
+          player.megaCredits -= 2;
+        }
```
Comment (vincentneko) [inline src/cards/venusNext/FloatingHabs.ts:41]: This if is not needed, the logic is handled by the addSelectHowToPayInterrupt.

File: src/cards/venusNext/FloatingHabs.ts [hunk 3]
```diff
@@ -29,30 +29,32 @@ export class FloatingHabs implements IActionCard,IProjectCard, IResourceCard {
...
-                  player.megaCredits -= htp.megaCredits;
-                  player.heat -= htp.heat;
-                  player.addResourceTo(foundCards[0], 1);
-                  return undefined;
-                }
-              );
+              game.addSelectHowToPayInterrupt(player, 2, false, false, "Select how to pay for Floating Habs action");
+            } else {
+              player.megaCredits -= 2;
             }
```
Comment (vincentneko) [inline src/cards/venusNext/FloatingHabs.ts:55]: Same here

**Pull request merged**